docs/basic-docs/atomicdex/atomicdex-api.md

```diff
@@ -725,19 +725,31 @@ Set the value of the `mm2` parameter in the [coins](../../../basic-docs/atomicde
 
 For terminal interface examples, see the examples section below.
 
+#### Using AtomicDEX Software on an Qtum Network
+
+The following information can assist the user/developer in connecting AtomicDEX software to the Qtum network:
```

Comment:
I think it will be a bit more accurate to write `... can assist the user/developer in using QRC20 tokens with AtomicDEX software on the Qtum network`